src/query/v2/procedure/mg_procedure_impl.cpp

```diff
@@ -2085,24 +2093,25 @@ mgp_error mgp_graph_is_mutable(mgp_graph *graph, int *result) {
   return mgp_error::MGP_ERROR_NO_ERROR;
 };
 
-mgp_error mgp_graph_create_vertex(struct mgp_graph *graph, mgp_memory *memory, mgp_vertex **result) {
-  return WrapExceptions(
-      [=] {
-        if (!MgpGraphIsMutable(*graph)) {
-          throw ImmutableObjectException{"Cannot create a vertex in an immutable graph!"};
-        }
-        auto vertex = graph->impl->InsertVertex();
-
-        auto &ctx = graph->ctx;
-        ctx->execution_stats[memgraph::query::v2::ExecutionStats::Key::CREATED_NODES] += 1;
-
-        if (ctx->trigger_context_collector) {
-          ctx->trigger_context_collector->RegisterCreatedObject(vertex);
-        }
-        return NewRawMgpObject<mgp_vertex>(memory, vertex, graph);
-      },
-      result);
-}
+//  TODO(jbajic) Fix Remove Gid
+// mgp_error mgp_graph_create_vertex(struct mgp_graph *graph, mgp_memory *memory, mgp_vertex **result) {
+//   return WrapExceptions(
+//       [=] {
+//         if (!MgpGraphIsMutable(*graph)) {
+//           throw ImmutableObjectException{"Cannot create a vertex in an immutable graph!"};
+//         }
+//         auto vertex = graph->impl->InsertVertex();
+
+//         auto &ctx = graph->ctx;
+//         ctx->execution_stats[memgraph::query::v2::ExecutionStats::Key::CREATED_NODES] += 1;
+
+//         if (ctx->trigger_context_collector) {
+//            ctx->trigger_context_collector->RegisterCreatedObject(vertex);
+//         }
+//         return NewRawMgpObject<mgp_vertex>(memory, nullptr, graph);
+//       },
+//       result);
+// }
```

Comment:
Why did you commented out the whole function here, while just part of the functions in other places? I would be consistent with that is possible.

Reply:
Because I get `static_assert failed due to requirement '!std::uses_allocator_v<mgp_vertex, memgraph::utils::Allocator<mgp_vertex>>' "Class declares std::uses_allocator but has no valid constructor overload. Refer to 'Uses-allocator construction' rules in C++ reference."` if commenting out vertex creation, not sure why I can just return `mgp_error` here